app/src/main/java/com/github/pockethub/android/ui/search/SearchUserListFragment.java

```diff
@@ -110,13 +117,14 @@ public void refresh() {
 
     @Override
     public void onListItemClick(ListView l, View v, int position, long id) {
-        final User result = (User) l.getItemAtPosition(position);
+        User result = (User) l.getItemAtPosition(position);
         ServiceGenerator.createService(getContext(), UserService.class)
                 .getUser(result.login())
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
-                .compose(this.<User>bindToLifecycle())
-                .subscribe(user -> startActivity(UserViewActivity.createIntent(user)));
+                .compose(this.bindToLifecycle())
+                .subscribe(
+                        response -> startActivity(UserViewActivity.createIntent(response.body())));
```

Comment:
Shouldn't `response ->` be on the same line as `subscribe` to keep consistent with coding style e.g. [here](https://github.com/pockethub/PocketHub/blob/60843d315f98599ad4946106e80d0eaf52827567/app/src/main/java/com/github/pockethub/android/util/PermissionsUtils.java#L45)?

I.e.

- `.subscribe(response ->`
2.                 `startActivity(UserViewActivity.createIntent(response.body())));`